### ISSUE TITLE
recognize multiple-episode files

### ISSUE THREAD

**Issue by eprenamer** [tvrenamer/tvrenamer#232]
It's fairly common to have two episodes (sometimes more) in a single file.

These may be identified like
* S06E06E07-HDTV-XviD-LOL.avi
* Friends-Season-05-Episode-23-24-The-One-in-Vegas.avi
* Star Trek The Next Generation Season 7 Episode 25 & 26 - All Good Things.avi
* The Golden Girls 03-18 & 19 Golden Moments Part.mkv

etc.

**Comment by daveharris** [comment]
Something I've run into is where both episodes of a pilot are shown at once and so is a single file, but it's technically part 1 and 2 of the pilot.
Something like [Lost Season 1](https://thetvdb.com/?seasonid=6345&seriesid=73739&tab=season), where it has "Pilot (1)" and "Pilot (2)". I don't know what the raw filename would be though ....
Not sure if there is something we can do about this, but just worth noting.

**Comment by eprenamer** [comment]
Right, that's exactly it, except there's no reason it needs to be a pilot.  "The Office" season 4 had five such episodes, including the first four of the season:

https://thetvdb.com/?tab=season&seriesid=73244&seasonid=28425&lid=7

How that gets translated into a file depends on the person creating the file.  Sometimes they might be broken up into individual episodes (parts 1 and 2).  But other times they're included in the same file, and given a filename like I describe in the OP.

**Comment by GrindLogic** [comment]
I'd like to update this thread with another possible markup that I've recently ran into:
```
WKRP S3E13, E14 (Dr. Fever and Mr. Tide).mp4
```
Which according to TVDB would be:
```
WKRP S3E13 - Dr. Fever and Mr. Tide (1)
WKRP S3E14 - Dr. Fever and Mr. Tide (2)
```
Which I personally would translate into:
```
WKRP 03x13-14 - Dr. Fever and Mr. Tide (1-2).mp4
```
Granted I'm fairly sure my MediaCenter app (Emby) would simply display it as:
```
13. Dr. Fever and Mr. Tide
```
